scripts/build-dist.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
-set -ex
+set -Exu
```

Comment:
What does capital E do? I don't see it on http://tldp.org/LDP/abs/html/options.html nor https://ss64.com/bash/set.html